### ISSUE TITLE
Table 6 Mislabeling Correction

### ISSUE THREAD

**Issue by KeithBonesJr** [cisagov/cyhy-system#94]
# 🐛 Summary #

Table 6 within the vulnerability scanning report shows count of vulns as "by host" when it is showing distinct occurrences. Column should specify occurrence rather than host. See attachment in linked Jira ticket.

## To reproduce ##

1. Download a weekly vulnerability scanning report.
2. Go to table 6

## Expected behavior ##

Table column should specify occurrence rather than host.

## Any helpful log output or screenshots ##

Please see JIRA ticket: https://jira.ceil.cyber.dhs.gov/browse/CYHYDEV-726 for screenshots
Not posting information here because it contains stakeholder specific information.

Because it is sorting these values in vulnerability_occurence. This is defined [here](https://github.com/cisagov/cyhy-reports/blob/a6862246f7342328fd55b72642b90c3d2b82819d/cyhy_report/customer/generate_report.py#L613-L625).

Count is never being set, so it is displaying as name, in this case "by host", this is where the change needs to happen.

**Comment by KeithBonesJr** [comment]
Line 608 needs to be tested which could be the issue.

**Comment by KeithBonesJr** [comment]
Line counts need to be verified. Will convert to console text and begin setting up virtual python env tomorrow.

**Comment by dav3r** [comment]
> Line counts need to be verified. Will convert to console text and begin setting up virtual python env tomorrow.

@KeithBonesJr A better way to reference code than line numbers or copy/pasting text is to use GitHub permalinks, like [this one](https://github.com/cisagov/cyhy-reports/blob/a6862246f7342328fd55b72642b90c3d2b82819d/cyhy_report/customer/generate_report.py#L613-L625).  If you are commenting in the same repository, permalinks will be rendered as a code block.

In this case, since this issue is in `cyhy-system` and the code is in `cyhy-reports`, the permalink URL below will not be rendered as a code block:
https://github.com/cisagov/cyhy-reports/blob/a6862246f7342328fd55b72642b90c3d2b82819d/cyhy_report/customer/generate_report.py#L613-L625

However, if this issue were to be moved into the the `cyhy-reports` repo (which I recommend), then we would see a code block.

Here is [GitHub's documentation about permalinks](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet).

**Comment by KeithBonesJr** [comment]
@dav3r  doesn't look like the cyhy-reports repo shows up for me. Is there access for me to be able to do that?

<img width="455" alt="image" src="https://github.com/cisagov/cyhy-system/assets/87027605/025e64ef-96ae-4dc1-93f6-0ba94860af5c">

**Comment by dav3r** [comment]
@KeithBonesJr I don't think you have that level of access, but if you give me a list of PRs that belong in the cyhy-reports repo, I'd be happy to transfer them there.
